README.md

```diff
@@ -0,0 +1,34 @@
+# XilDebug
+
+[<img src="https://travis-ci.org/xil-se/xildebug.svg?branch=master">](https://travis-ci.org/xil-se/xildebug)
+
+**XilDebug** is a piece of hardware that can act as a CMSIS-DAP compliant debugger, UART bridge and power profiler all in one package. This repository contains the firmware.
+
+# Building
+
+Make sure you have a recent gcc-arm-none-eabi toolchain. 7.2.1 is known to work.
+
+`make` in the root builds the firmware.
+
+# Debugging/flashing
+
+Install a recent version of `openocd`.
+
+Connect a debugger (e.g. an STLink or a DAP-link compliant debugger) to the SWD pins.
```

Comment:
CMSIS-DAP not DAP-link

Reply:
is fixed.. i amended and force pushed, may be i broke github now